db/seeds.rb

```diff
@@ -1 +1,20 @@
+require 'faker'
 
+1.times do
+  User.create!(
+    email: Faker::Internet.email,
+    encrypted_password: Faker::Internet.password
```

Comment:
First, I installed the `faker` gem, then I want to seed it with a user.  Just one for now, so I created a loop to create `User`; the email and the encrypted_password will be generated by faker.  I used "encrypted_password" because that's the name of password in my `schema.rb`.